src/Text/Pandoc/Readers/Odt/ContentReader.hs

```diff
@@ -190,6 +206,22 @@ popStyle =     keepingTheValue (
 getCurrentListLevel :: OdtReaderSafe _x ListLevel
 getCurrentListLevel = getExtraState >>^ currentListLevel
 
+--
+updateMediaWithResource :: OdtReaderSafe (FilePath, B.ByteString) (FilePath, B.ByteString)
+updateMediaWithResource = keepingTheValue (
```

Comment:
The indentation looks really odd, is this really intended? Can't most of this ago onto a single line?